tensorflow/core/util/rocm_solvers.cc

```diff
@@ -862,32 +862,7 @@ static inline Status TrsmImpl(GpuExecutor* gpu_executor, SolverFnT solver,
 
 TF_CALL_LAPACK_TYPES_NO_COMPLEX(TRSM_INSTANCE);
 
-<<<<<<< HEAD
 #define TRSM_BATCHED_INSTANCE(Scalar, type_prefix)                            \
-=======
-template <typename Scalar, typename SolverFnT>
-Status MatInvBatchedImpl(GpuExecutor* gpu_executor, SolverFnT solver,
```

Comment:
Why remove MatInvBatchedImpl from the upstream branch here?

Reply:
I didn't remove it.  That was caused by a git merge error.